gravitee-apim-console-webui/src/management/api/proxy/properties-ng/api-properties-old.component.ts

```diff
@@ -53,7 +53,7 @@ export class ApiPropertiesComponent implements OnInit, OnDestroy {
 
   constructor(
     private readonly apiService: ApiV2Service,
-    private readonly apiPropertiesService: ApiPropertiesService,
+    private readonly apiPropertiesService: ApiPropertiesOldService,
```

Comment:
Do we rename `private readonly apiPropertiesService: ApiPropertiesOldService,` to `private readonly apiPropertiesOldService: ApiPropertiesOldService,` too?

Reply:
I think it's very important here. At the end, this component will be deleted once the migration is done.